gordo_components/model/factories/feedforward_autoencoder.py

```diff
@@ -86,10 +98,14 @@ def feedforward_model(
         model.add(Dense(units=units, activation=activation))
 
     # Final output layer
```

Comment:
This comment doesn't seem relevant to this line, should probably move it below this conditional.